src/components/Navbar.jsx

```diff
@@ -3,41 +3,79 @@ import { Menu, Header, Icon, Responsive, Sidebar } from 'semantic-ui-react'
 import '../css/navbar.css'
 import { NavLink } from 'react-router-dom'
 import { connect } from 'react-redux'
+import Login from './Login'
 import Logout from './Logout'
 
 class Navbar extends Component {
-  state = { visibleSidebar: false }
+  state = { visibleSidebar: false, modalOpen: false }
+
+  handleModalOpen = () => {
+    this.setState(prevState => {
+      return {
+        modalOpen: !prevState.modalOpen
+      }
+    })
+  }
 
   handleShowClick = () => this.setState({ visibleSidebar: true })
   handleSidebarHide = () => this.setState({ visibleSidebar: false })
 
   render() {
     const notMobile = { minWidth: Responsive.onlyMobile.maxWidth + 1 }
-    let logOut, logIn
+    let logOut, logIn, welcomeMessage, createRecipe
 
     if (this.props.currentUser.isSignedIn) {
+      welcomeMessage = (
+        <Menu.Item style={{ backgroundColor: 'white' }}>
+          <Header
+            position='right'
+            style={{ color: 'green' }}
+            id="welcome-message"
+          >
+            Hello {this.props.currentUser.attributes.name}
+          </Header>
+        </Menu.Item>
+      )
       logOut = (
-        <Logout />
+        <Logout
+          handleModalOpen={this.handleModalOpen}
+        />
+      )
+      createRecipe = (
+        <Menu.Item id='nav-create' as={NavLink} to='/create'>
+          <Header position='right'>
+            Create Recipe
+          </Header>
+        </Menu.Item>
       )
     } else {
       logIn = (
-        <Menu.Item id='nav-login' as={NavLink} to='/login'>
-          <Header position='right' style={{ fontFamily: 'Condiment' }}>
-            Login
+        <Menu.Item id='nav-login' className='fake-link-hover'>
+          <Header
+            position='right'
+            onClick={this.handleModalOpen}>
+            Log in
           </Header>
+          <Login
+            modalOpen={this.state.modalOpen}
+            handleModalOpen={this.handleModalOpen}
+          />
         </Menu.Item>
       )
     }
 
     return (
       <>
         <Responsive {...Responsive.onlyMobile}>
-          <Menu id='navbar' style={{ marginTop: "1em" }}>
-            <Menu.Item>
+          <Menu id='navbar' style={{ marginTop: "1em" }} borderless={true}>
+            <Menu.Item
+              as={NavLink}
+              to='/'
+              id='nav-home'>
               <Header
                 position='left'
-                id='navbar-header'
-                style={{ fontSize: '2rem', textAlign: 'center', fontFamily: 'Anton' }}>
+                className='navbar-header'
+                style={{ fontSize: '2rem', textAlign: 'center', fontFamily: 'Abril Fatface', fontWeight: 'bold' }}>
```

Comment:
Why do you use css classes AND inline styling?

Reply:
Forgot to remove it, thank you for pointing it out!